contents/docs/posthog-ai/pricing-faq.mdx

```diff
@@ -0,0 +1,62 @@
+# PostHog AI credits
+
+PostHog uses **AI credits** to charge for all of its AI features.  
+Each time you interact with an AI-powered feature — like asking PostHog to write SQL, summarize data, or analyze sessions — a small number of credits are used based on the complexity of your request.
+
+This means you only pay for what you actually use.  
+There’s no fixed subscription — and every organization has a free tier worth $25 (2,500 credits) so you can explore all PostHog’s AI features before paying anything.
+
+### **When AI credits are used**
+
+AI credits are consumed whenever PostHog performs intelligent work for you.  
+You’ll spot these features by the ✨ icon or when using the in-app chat.
+
+Examples include:
+- Searching for session recordings in natural language  
+- Summarizing experiment results  
+- Writing or debugging SQL with AI  
+
+### **What doesn’t use credits**
+
+Getting help on how to use PostHog never costs credits.  
```

Comment:
I don't think this is _strictly_ true, i.e. if you getting help involves a query being created – that costs credits.

Because we have a (we could say generous) free tier, is it perhaps then simpler – for both the user an us – to charge for the root node always? Regardless of whether just the docs search tool was involved, insight creation, or no tools at all. Especially with the new agent orchestration based on mode-switching, being prototyped by @skoob13, the tools become "dumb", and it's even trickier to say "this is just getting help" vs. "this is doing some work for the user".